resource-validator/src/main/scala/com/broadinstitute/dsp/resourceValidator/DataprocWorkerChecker.scala

```diff
@@ -40,7 +40,10 @@ object DataprocWorkerChecker {
             .getCluster(runtime.r.googleProject, regionName, DataprocClusterName(runtime.r.runtimeName))
           runtime <- clusterOpt.flatTraverse { c =>
             val doesPrimaryWorkerMatch =
-              runtime.workerConfig.numberOfWorkers.getOrElse(0) == c.getConfig.getWorkerConfig.getNumInstances
+              if (c.getConfig.getWorkerConfig.getNumInstances == 0)
```

Comment:
we need to add to the if-clause 

```scala
runtime.r.status.toLowerCase == "stopped"
```

since we want to ignore the zero preemptible case only if the cluster was stopped because we remove preemptibles before stopping the cluster

Reply:
Also, the check above needs to go into the 

```scala
val doesSecondaryWorkerMatch = ...
```

block instead, no? Preemptibles are secondary workers.

Reply:
yep...this actually caused unit tests...posted the PR too soon :)